Brand/NCBrand.swift

```diff
@@ -110,9 +110,14 @@ final class NCBrandOptions: @unchecked Sendable {
                 enforce_passcode_lock = (str as NSString).boolValue
             }
         }
-#if DEBUG
+        #if DEBUG
         pushNotificationServerProxy = "https://c0004.customerpush.nextcloud.com"
-#endif
+        #else
+        if pushNotificationServerProxy.isEmpty,
+            brand == "Nextcloud" {
+            pushNotificationServerProxy = "https://push-notifications.nextcloud.com"
```

Comment:
Hm. But this means for branded clients without their own push proxy, we still try to connect to our server.
And that is wrong. Since this will not work, we should not do it.

I think it is better to change the logic, that only a push connection is made, when there is a server (pushNotificationServerProxy.isNotEmpty)